api/buckets/pb/buckets.proto

```diff
@@ -179,8 +179,43 @@ message PullPathAccessRolesResponse {
     map<string, PathAccessRole> roles = 1;
 }
 
+message ArchiveConfig {
+    int32 rep_factor = 1;
+	int64 deal_min_duration = 2;
+	repeated string excluded_miners = 3;
+	repeated string trusted_miners = 4;
+	repeated string country_codes = 5;
+	ArchiveRenew renew = 6;
+	string addr = 7;
+	uint64 max_price = 8;
+	bool fast_retrieval = 9;
+	int64 deal_start_offset = 10;
+}
+
+message ArchiveRenew {
+	bool enabled = 1;
+	int32 threshold = 2;
+}
+
+message DefaultArchiveConfigRequest {
+    string key = 1;
+}
+
+message DefaultArchiveConfigResponse {
+    ArchiveConfig archive_config = 1;
+}
+
+message SetDefaultArchiveConfigRequest {
+    string key = 1;
+    ArchiveConfig archive_config = 2;
+}
+
+message SetDefaultArchiveConfigResponse {
+}
+
 message ArchiveRequest {
     string key = 1;
+    ArchiveConfig archive_config = 2;
```

Comment:
Added the custom override config here. This can be `nil` when the default should be used.